index.php

```diff
@@ -409,9 +482,96 @@ function initMap() {
   <script src="assets/vendor/waypoints/jquery.waypoints.min.js"></script>
   <script src="assets/vendor/counterup/counterup.min.js"></script>
   <script src="assets/vendor/aos/aos.js"></script>
+ <!--  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+   -->
+  
 
   <!-- Template Main JS File -->
   <script src="assets/js/main.js"></script>
+  <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script> -->
+  <script>
+      
+$(document).ready( function () {  
+
+        const userId=<?php echo($_SESSION['user_id']); ?>;
+        
+        //Lock Input Field By Default as Modal is Opened    
+        $("#my-profile").click(function () {
+          $("input").prop('disabled', true);
+          $("input").css('background-color','antiquewhite');
+          // $(this).prev().css("background-color","antiquewhite");
+          $("#apply-changes-btn").css({'background-color':'#6c757d','font-weight':'500','cursor': 'not-allowed'});
+          
+          //Fetch from Database using ajax
+          
+        /*   $.ajax({
+                  url: "fetch-user.php",
+                  method:"POST",
+                  data:{userId:userId},
+                  success:function(response)
+                  {
+                    //Encode response into array
+                    arr=JSON.parse(response);
+                    
+                    //Insert the fetched data into modal input fields
+                    $("#full-name").val(arr[0]["name"]);
+                    $("#email").val(arr[0]["email"]);
+                    $("#organisation").val(arr[0]["organisation"]);
+                    $("#state").val(arr[0]["state"]);
+                  }
+                });
+ */
```

Comment:
Code to fetch data, if modal opened. It's been commented now due to database non availability.